README.md

```diff
@@ -3,6 +3,7 @@
 <!-- badges: start -->
 [![Project Status: Concept – Minimal or no implementation has been done yet, or the repository is only intended to be a limited example, demo, or proof-of-concept.](https://www.repostatus.org/badges/latest/concept.svg)](https://www.repostatus.org/#concept)
 [![Build Status](https://travis-ci.com/mrc-ide/dust.svg?branch=master)](https://travis-ci.com/mrc-ide/dust)
+[![AppVeyor build status](https://ci.appveyor.com/api/projects/status/github/mrc-ide/dust?branch=master&svg=true)](https://ci.appveyor.com/project/mrc-ide/dust)
 [![CodeFactor](https://www.codefactor.io/repository/github/mrc-ide/dust/badge)](https://www.codefactor.io/repository/github/mrc-ide/dust)
```

Comment:
Is this (and codecov) C and R, or just R?

Reply:
codecov is both C and R, codefactor will probably start screening the C at some point, though it does not seem to have started yet